sprout/core/write_file.py

```diff
@@ -0,0 +1,16 @@
+from pathlib import Path
+
+
+def write_file(path: Path, string: str) -> Path:
+    """Writes a file to the given path with the given content.
```

Comment:
```suggestion
def write_file(string: str, path: Path) -> Path:
    """Writes a file to the given path with the given content.
```

When writing, the path should always be after the string, so as to pipe the output into the function's first position (as is done in R's `|>` and in bash's `|`). I know Python doesn't really have a pipe (😢) but it's good to think in that framing.